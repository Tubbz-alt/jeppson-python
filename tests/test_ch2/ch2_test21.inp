    0.1016   0.0141   45.72     1.131E-6    2.1E-6   9.81   
