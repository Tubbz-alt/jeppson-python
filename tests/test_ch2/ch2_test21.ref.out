Q=    0.0141D=    0.1016L=     45.72F=   0.01657HEADLOSS=    1.1493
